scripts/checkfornewdemuxonhasta.bash

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+shopt -s expand_aliases
+source ${HOME}/.bashrc
+useprod
+
+########
+# VARS #
+########
+
+#ERROR_EMAIL=clinical-demux@scilifelab.se
```

Comment:
Sure we shouldn't just get these all of us?

Reply:
Yes of course!